exp/runtime/hooks/api/v1alpha1/topologymutation_types.go

```diff
@@ -73,21 +78,25 @@ type GeneratePatchesResponse struct {
 	CommonResponse `json:",inline"`
 
 	// items is the list of generated patches.
+	// +required
```

Comment:
optional + omitempty

Folks don't have to send patches back if they don't want to patch anything